pkg/apis/migration/v1alpha1/condition.go

```diff
@@ -357,3 +358,9 @@ func (r *Conditions) SetReconcileFailed(err error) {
 	})
 	r.EndStagingConditions()
 }
+
+func (r *Conditions) Log(log logging.Logger) {
+	for _, cond := range r.List {
+		log.Info("updated condition", cond)
```

Comment:
I would prefer this instead in favour of adding the entier condition object for readability of the logs, but not too opinionated if folks think otherwise makes more sense

Reply:
Can you give an example of what both formats (single log per condition vs all together) look like so we can get an idea?

Reply:
```
{“info”, “kube-builder”, “all the other default fields”, “updated conditions”, [{“condition1 entire json”}, {“condition2 entire json”}]}
vs
{“info”, “kube-builder”, “all the other default fields”, “updated condition”, {“condition1 entire json”}}
{“info”, “kube-builder”, “all the other default fields”, “updated condition”, {“condition2 entire json”}}
```

I have not built the image but conceptually I am thinking the difference would be as above

Reply:
I think I tend to agree with you that one log per condition will be more readable.